.gitignore

```diff
@@ -1,3 +1,5 @@
+arek.md
```

Comment:
wywalić takie śmieci w ogóle z projektu (przerzucić do innego folderu chociażby), gitignore to nie jest lista śmieci z twojego komputera